crmprtd/process.py

```diff
@@ -0,0 +1,78 @@
+import sys
+from importlib import import_module
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import logging
+from argparse import ArgumentParser
+
+from crmprtd.align import align
+from crmprtd.insert import insert
+from crmprtd import logging_args, setup_logging
+
+
+def process_args(parser):
+    parser.add_argument('-c', '--connection_string',
+                        help='PostgreSQL connection string')
+    # FIXME: I do not think this arg gets used anywhere
+    parser.add_argument('-D', '--diag',
+                        default=False, action="store_true",
+                        help="Turn on diagnostic mode (no commits)")
+    parser.add_argument('--sample_size', type=int,
+                        default=50,
+                        help='Number of samples to be taken from observations '
+                             'when searching for duplicates '
+                             'to determine which insertion strategy to use')
+    parser.add_argument('-N', '--network',
```

Comment:
We should add the "choices" argument here, so the user knows what networks are available. For example *I* can't even remember whether I need to say `-N ec` or `-N EC_raw`, so guidance would be helpful.